packages/inter-protocol/src/auction/scheduler.js

```diff
@@ -199,12 +196,12 @@ export const makeScheduler = async (
         advanceRound();
         break;
       case 'endExactly':
-        if (advanceRound()) {
-          finishAuctionRound();
-        }
+        // do both the "during" and "after" behaviors
```

Comment:
_comment_:

With the context of this bug it makes so much sense. Of course we always wanna finish an auction after `endExactly`.